starter-code/css/main.css

```diff
@@ -0,0 +1,88 @@
+
+* {
+  font-family: Lato;
+  font-weight: 300;
+}
+
+body {
+  margin: 75px 50px;
+}
+
+h1 a {
+  font-size: 36pt;
+  text-decoration: none;
+  color: #f8cd84;
+}
+
+header nav a {
+  color: #e5e6ed;
+  text-decoration: none;
+  border-bottom: 1px solid;
+  padding-bottom: 3px;
+  margin: 0 7px;
+  font-size: 15pt;
+}
+
+header nav a:hover, header nav a:focus {
+  color: #f8cd84;
+}
+
+article {
+  max-width: 650px;
+  border-bottom: 1px solid;
+  border-color: #e5e6ed;
+  margin: 70px 0;
+}
+
+article h3 {
+  font-size: 24pt;
+  color: #e5e6ed;
+  margin-bottom: 0;
+}
+
+h2 a {
+  color: black;
+  text-decoration: none;
```

Comment:
You've started to repeat this property over and over throughout the file. Consider creating an anchor class at the beginning of the file that clears this text-decoration to help keep your code DRY.